src/figuresClasses.ts

```diff
@@ -1,11 +1,77 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: string = 'triangle';
+
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    const NotPositive = a <= 0 || b <= 0 || c <= 0;
+    const isNotTriangle = a + b <= c || a + c <= b || b + c <= a;
+
+    if (NotPositive || isNotTriangle) {
+      throw new Error('incorrect value');
+    }
+    this.color = color;
+  }
+
+  getArea(): number {
+    const s = (this.a + this.b + this.c) / 2;
```

Comment:
We need more specific name for values, not just "s"
```suggestion
    const semiperimeter = (this.a + this.b + this.c) / 2;
```